### PR TITLE
Handle class name inflector

### PR DESCRIPTION
**Pull request by irfanshehzadsandhu**
Add HandleClassNameInflector as mentioned in Tactician simple command bus

### CODE REVIEW

File: src/handler/MethodNameInflector/HandleClassNameInflector.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import MethodNameInflector from './MethodNameInflector';
+
+export default class HandleClassNameInflector extends MethodNameInflector {
+    constructor(methodName) {
+        super();
+        this.methodName = methodName || "handle";
```
Comment (erickjth) [inline src/handler/MethodNameInflector/HandleClassNameInflector.js:6]: According to this class, https://github.com/thephpleague/tactician/blob/master/src/Handler/Mapping/MethodName/HandleLastPartOfClassName.php, seems like it does not need the property `this.methodName`, so please remove and use `handle` hardcoded in the `inflect` method.

**Review by erickjth — COMMENTED** [review]
Thanks for sharing this PR. I just left a small comment.

File: src/handler/MethodNameInflector/HandleClassNameInflector.js (hunk 1 above)
Comment (irfanshehzadsandhu) [reply src/handler/MethodNameInflector/HandleClassNameInflector.js]: Removed this.methodName and used handle hardcoded

Comment (erickjth) [reply src/handler/MethodNameInflector/HandleClassNameInflector.js]: Cool, Thanks!

**Review by erickjth — APPROVED** [review]

**Pull request merged**